### PR TITLE
Fe profile listings page

### PR DESCRIPTION
**Pull request by quddd**
-----------------------------------------------------Shows a list of profiles----------------------------------------------
Screen Shot 2021-01-20 at 9 28 53 AM
---------------------------Responsive-----------------------------------------------------------------------------------
Screen Shot 2021-01-20 at 9 28 32 AM

-------------------------------------- Only Show Sitters -----------------------------------------------------------------
Screen Shot 2021-01-20 at 9 29 14 AM

### CODE REVIEW

**Review by dproc96 — COMMENTED** [review]
This looks good except for a couple of minor nitpicks I noticed:
- In mobile view, the profile cards aren't centered, they're off to the left
- A profile card with a description is a different size than a profile card without one
- In profile listings, the user can see themselves if they are a sitter
- The spacing looks off in some of your files, be sure to run prettier before committing

File: client/src/pages/ProfileListings.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+            {sitters.error}
+        </h1>
+    }
+    if(sitters.loading){ // display a loading page while page is requesting data
+        content = <h1>Loading...</h1>
+    }
+    if(sitters.data){ // set profile data to display
+        content = 
+        sitters.data.map((sitter, key) => 
+            <ProfileCard
```
Comment (dproc96) [inline client/src/pages/ProfileListings.js]: Be sure to add a key attribute to this

**Review by dproc96 — APPROVED** [review]
This looks good but just make two minor changes before merging:
1. Run Prettier on profileCard.js, searchAndFilter.js, and ProfileListings.js
2. Rename profileCard.js and searchAndFilter.js to capitalize the first letter since they're components

**Comment by quddd** [comment]
****Fixed** Text overflow**

**Pull request closed without merging**